mteb/tasks/CLSD/WMT1921/CrossLingualSemanticDiscriminationWMT19.py

```diff
@@ -0,0 +1,129 @@
+from typing import Dict, List
+
+import datasets
+
+from mteb.abstasks import AbsTaskCLSD, CrosslingualTask, TaskMetadata
+
+_LANGUAGES = {
+    "wmt19.de.fr": ["deu-Latn", "fra-Latn"],
+    "wmt19.fr.de": ["fra-Latn", "deu-Latn"],
+}
+
+
+def _build_lang_pair(langs: List[str]) -> str:
+    """Builds a language pair separated by a dash.
+    e.g., ['eng-Latn', 'deu-Latn'] -> 'eng-deu'.
+    """
+    return langs[0].split("-")[0] + "-" + langs[1].split("-")[0]
+
+
+def extend_lang_pairs() -> Dict[str, List[str]]:
+    eval_langs = {}
+    for langs in _LANGUAGES.values():
+        lang_pair = _build_lang_pair(langs)
+        eval_langs[lang_pair] = langs
+    return eval_langs
+
+
+_EVAL_LANGS = extend_lang_pairs()
+
+
+class CrossLingualSemanticDiscriminationWMT19(AbsTaskCLSD, CrosslingualTask):
+    metadata = TaskMetadata(
+        name="CrossLingualSemanticDiscriminationWMT19",
+        dataset={
+            "path": "Andrianos/clsd_wmt19_21",
+            "revision": "9627fbdb39b827ee5c066011ebe1e947cdb137bd",
+        },
+        description="Evaluate a multilingual embedding model based on its ability to discriminate against the original parallel pair against challenging distractors - spawning from WMT19 DE-FR test set",
+        reference="https://huggingface.co/datasets/Andrianos/clsd_wmt19_21",
+        type="CLSD",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=_EVAL_LANGS,
+        main_score="accuracy",
+        date=("2018-01-01", "2023-12-12"),
+        form=["written"],
+        domains=["News"],
+        task_subtypes=[],
+        license="CC BY-SA 4.0",
+        socioeconomic_status="high",
+        annotations_creators="derived",
+        dialect=[],
+        text_creation="LM-generated and verified",
+        bibtex_citation="to_update",
```

Comment:
missing

Reply:
Will add it once the preprint is available, I added a reminder 👍

Reply:
Let us just keep it as an empty string or now with a comment that a preprint is coming

Reply:
Updated